src/devices/Card/ErrorType.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Card
+{
+    /// <summary>
+    /// The type of error
+    /// </summary>
+    public enum ErrorType
+    {
+        ProcessCompletedNormal,
```

Comment:
should the first one be called Success for easier discovery?

Reply:
I added ```Success``` then at the end with the same value as ```PorcessCompletedNormal```.